stdlib/contrib/mhall119/helpers/helpers.flux

```diff
@@ -0,0 +1,94 @@
+// Package helpers provides shortcut functions for small common tasks
+//
+// ## Metadata
+// introduced: ??
+package helpers
+
+import "array"
+import "json"
+// yieldValue lets you display a scalar in the InfluxDB UI by
+// wrapping it in an Array
+//
+// ## Parameters
+// - v: Input data.
+// - name: Name to pass to the subsequent yield() call.  Default is 'value'.
+//
+// ## Examples
+//
+// ### Yield an option value
+// ```no_run
+// import "contrib/mhall119/helpers"
+//
+// option testData = {testValue: 125}
+// yieldValue(v: testData.testValue)
+// ```
+//
+// ## Metadata
+// tags: 
+//
+defaultYieldValName="value"
+yieldValue = (v, name=defaultYieldValName) =>
+    array.from(
+      rows: [
+          {_time: now(), _value: v},
+      ]
+    )
+    |> yield(name: name)
+
+// yieldObject lets you display an object in the InfluxDB UI by
+// encoding it as String
+//
+// ## Parameters
+// - o: Object to yield
+// - name: Name to pass to the subsequent yield() call.  Default is 'object'.
```

Comment:
```suggestion
// - r: Record to yield.
// - name: Yield name. Default is `record`.
```

Reply:
This function is meant to work with any Flux object, not just records returned by a query. Unless "Record" is the term used to describe any data type that's not a table or a scalar.

Reply:
I see. These are known as [composite types](https://docs.influxdata.com/flux/v0.x/data-types/composite/) - records, arrays, dictionaries, and functions. And really, with that context, this function can display any Flux type, basic, composite, or regexp. The caveat is that it always gets output as a string.

If the purpose of this package is simply to be able to show something in the single stat viz, this might be the only function you need. I guess the downside would be losing number truncation in the UI.